src/librustdoc/html/static/main.js

```diff
@@ -951,3 +951,8 @@
     }());
 
 }());
+
+// Sets the focus on the search bar at the top of the page
+function focusSearchBar() {
+    document.getElementsByName('search')[0].focus();
```

Comment:
Oh, I didn't realize this hadn't landed yet... anyway it's in a roll up now, so I'm not going to touch it.